citesphere/src/main/java/edu/asu/diging/citesphere/core/util/model/impl/CitationHelper.java

```diff
@@ -122,6 +123,18 @@ public void updateCitation(ICitation citation, CitationForm form, IUser user) {
             }
         }
     }
+    
+    @Override
+    public void updateCitation(ICitation citation, String collection, IUser iUser) {
```

Comment:
this should have a name that reflects what this method is doing (adding a collection) and javadoc.

Reply:
Changed name and added Javadoc